File: fungo_zumbi/fungo_zumbi.cpp
#include <iostream>
using namespace std;

int main(){
    int tempo;
    char estado;
    cin >> estado >> tempo;
    if (estado=="S"){
        if (tempo>=30){
            cout << "Imune! Siga para um local seguro";
        }
        else if( 7 <= tempo && tempo <= 30){
            cout << "Acesso negado! Fique em observacao";
        }
        else{
            cout << "Acesso negado! Isolamento urgente";
        }
    }
    else{
        cout << "Acesso permitido!";
    }     
}

File: fungo_zumbi/Fungo_Zumbi.py
#Detectar a infecção
#Detectar o número de dias da infecção
# if x>=30 = imune
# if 7<=x<=30 = observação
# if x<7 = isolada

#S = infectada
#N = não infectada

entrada = input()
estado = entrada[0]
tempo = int(entrada[1:])
if estado=="S":
    if tempo>=30:
        print("Imune! Siga para um local seguro")
    elif 7 <= tempo <= 30:
        print("Acesso negado! Fique em observacao")
    else:
        print("Acesso negado! Isolamento urgente")
else:
    print("Acesso permitido!")